### ISSUE TITLE
Accessibility: A11y_Live Chat Widget_Latest Bot Chat Window_Screenreader: After sending any available commands screen reader announce unnecessary information.

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#3052]
Copied from #1750248.

[1750248_A11y_Live Chat Widget_Latest Bot Chat Window_Screenreader.zip](https://github.com/microsoft/BotFramework-WebChat/files/4397024/1750248_A11y_Live.Chat.Widget_Latest.Bot.Chat.Window_Screenreader.zip)

“Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.” 

 

Environment Details:

Application Name: Live Chat Wedget

#URL:https://xxx.blob.core.windows.net/portals/democ2adaptivecard.html

Browser Versions: Microsoft Edge Version 80.0.361.69 (Official build) (64-bit)

Windows Version: Windows10

Chrome Version: Version 80.0.3987.149 (Official Build) (64-bit)

 

 

Repro Steps: ​​

1)Launch the application url : https://xxx.blob.core.windows.net/portals/democ2adaptivecard.html

2)Latest Bot icon Button get open and Click on it.

3)"Latest Bot Chat Window" screen get open.

4)Start NVDA and Send available commands.

Actual:

 

After sending any available commands screen reader announce unnecessary information.

 

Same issue is observed with edge+narrator.

Expected:

It should only reads received message information Not whole description and unneccessary information.

User Impact:

Screen reader users will face difficulty as screen reader unnecessarily announces long description.

Recommendations: ​

Refer below link which is repository of bug fixes code snippets:

https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

 

MAS Reference:

MAS1.3.1:

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame2.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Comment by compulim** [comment]
This should be fixed in #3076.
